source/builders.txt

```diff
@@ -0,0 +1,305 @@
+.. _php-builders:
+
+========================
+Operations with Builders
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: aggregation, query, code example, type-safe
+
+Overview
+--------
+
+In this guide, you can learn about the **builder classes**
+that the {+library-short+} provides to create types used in your
+operations. You can use the builder classes and factory methods from the
+:ref:`php-aggregation-builder-api` feature to create filters for other
+operations such as find, update, and delete operations.
```

Comment:
```suggestion
operations. You can use the builder classes and factory methods from the
aggregation builder feature to create filters for other
operations such as find, update, and delete operations. To learn more about aggregation builders,
see the :ref:`php-aggregation-builder-api` section of the {+library-short+}'s aggregation guide. 
```
S: Include link intro text
style guide: "Link text should begin with the purpose of the cross-reference"